src/main/java/Main.java

```diff
@@ -120,8 +120,8 @@ public static void main(String[] args) throws Exception {
             String v = System.getProperty("java.class.version");
             if (v!=null) {
                 try {
-                    if (Float.parseFloat(v)<52.0f)
-                        throw new UnsupportedClassVersionError(v);
+                    if (Float.parseFloat(v)!=52.0f)
```

Comment:
Shouldn't it rather be something like: 

```java
if( Math.abs(Float.parseFloat(v) - 52.0f) < 0.001f )
```
?

Reply:
Should not matter. Float comparison is safe enough due to internal margins. Well, it weird but safe